java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -29,46 +44,267 @@ public void setUp() {
         service = new RedisClient(connectionManager, commandManager);
     }
 
+    @SneakyThrows
     @Test
-    public void customCommand_success() throws ExecutionException, InterruptedException {
+    public void customCommand_returns_success() {
         // setup
         String key = "testKey";
         Object value = "testValue";
         String cmd = "GETSTRING";
+        String[] arguments = new String[] {cmd, key};
         CompletableFuture<Object> testResponse = mock(CompletableFuture.class);
         when(testResponse.get()).thenReturn(value);
-        when(commandManager.submitNewCommand(any(), any())).thenReturn(testResponse);
+
+        // match on protobuf request
+        when(commandManager.submitNewCommand(eq(CustomCommand), eq(arguments), any()))
+                .thenReturn(testResponse);
 
         // exercise
-        CompletableFuture<Object> response = service.customCommand(new String[] {cmd, key});
+        CompletableFuture<Object> response = service.customCommand(arguments);
         String payload = (String) response.get();
 
         // verify
         assertEquals(testResponse, response);
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
     @Test
-    public void customCommand_interruptedException() throws ExecutionException, InterruptedException {
+    public void customCommand_throws_InterruptedException() {
```

Comment:
I propose to test with another type of exception
Maybe move to exception handling? (is it already covered though?)

Reply:
I don't see any InterruptedException handling in `ExceptionHandlingTests`.

Reply:
removed